bin/compile

```diff
@@ -69,11 +69,14 @@ RUNTIME_FILE="runtime.txt"
 if [ -f "$RUNTIME_FILE" ] ; then
     log "$RUNTIME_FILE found, skipping $RUNTIME_FILE generation" >&2
     exit 0
+elif [ -f .python_version ]; then 
```

Comment:
Hi! The filename is `.python-version` rather than `.python_version` :-)

The name isn't something specific to Heroku, but a common convention lots of tools use, eg:

- https://github.com/pyenv/pyenv?tab=readme-ov-file#understanding-python-version-selection
- https://docs.astral.sh/uv/guides/projects/#python-version

Reply:
ah lol

Reply:
fixed, thanks

Reply:
Thanks @edmorley